src/bundle/Resources/views/themes/admin/content/form_fields.html.twig

```diff
@@ -49,7 +49,10 @@
 
     {% set label_attr = label_attr|merge({'class': (label_attr.class|default('') ~ ' ibexa-label ibexa-data-source__label ibexa-data-source__label')|trim}) %}
     {% set label_wrapper_attr = label_wrapper_attr|default({})|merge({'class': (label_wrapper_attr.class|default('') ~ ' ibexa-data-source__label-wrapper ibexa-data-source__label-wrapper')|trim}) %}
-    {% set attr = attr|merge({'class': (attr.class|default('') ~ ' ibexa-data-source__input ibexa-data-source__input ibexa-input--small')|trim}) %}
+    {% set attr = attr|merge({
+        class: (attr.class|default('') ~ ' ibexa-data-source__input ibexa-data-source__input ibexa-input--small')|trim,
+        is_small: true,
```

Comment:
I don't like passing this in `attr` because we usually use `attr` as HTML element prams, but this is how it is currently done in the dropdown widget:
https://github.com/ibexa/admin-ui/blob/034e14592cbbc7e4044723c22467de9c0936039a/src/bundle/Resources/views/themes/admin/ui/form_fields/dropdown_widget.html.twig#L25
Another option would be to deprecate this approach and add a separate `is_small` variable.

Reply:
Maybe you have other ideas?